dist/_locales/ko/messages.json

```diff
@@ -0,0 +1,232 @@
+    "description": ""
+  },
+  "ago": {
+    "message": "전",
+    "description": ""
+  },
+  "search_tabs": {
+    "message": "탭 검색",
+    "description": ""
+  },
+  "no_results_found": {
+    "message": "결과 없음",
+    "description": ""
+  },
+  "bookmark": {
+    "message": "북마크",
+    "description": ""
+  },
+  "copy_url": {
+    "message": "URL 복사",
+    "description": ""
+  },
+  "pin": {
+    "message": "핀 고정",
+    "description": ""
+  },
+  "screenshot_visible_area": {
+    "message": "보이는 영역 캡쳐",
+    "description": ""
+  },
+  "duplicated": {
+    "message": "중복됨",
+    "description": ""
+  },
+  "tab_cleaner_header": {
+    "message": "사용하지 않는 탭 자동 종료",
+    "description": ""
+  },
+  "tab_cleaner_description": {
+    "message": "이 설정을 활성화하면, 일정 기간 이상 사용하지 않은 탭들이 자동으로 종료됩니다.",
+    "description": ""
+  },
+  "duration": {
+    "message": "기간",
+    "description": ""
+  },
+  "unit": {
+    "message": "단위",
+    "description": ""
+  },
+  "duration_unit_day": {
+    "message": "일",
+    "description": ""
+  },
+  "duration_unit_hour": {
+    "message": "시간",
+    "description": ""
+  },
+  "popup_size_header": {
+    "message": "팝업 창 크기 조절",
+    "description": ""
+  },
+  "popup_size_description": {
+    "message": "높이 범위: 200-600px, 너비 범위: 400-800px.",
+    "description": ""
+  },
+  "popup_element_scale_header": {
+    "message": "Popup element scale",
+    "modified_message": "Adjust the size for font and icons",
+    "message": "텍스트 및 아이콘 크기 조절",
+    "description": ""
+  },
+  "popup_element_scale_description": {
+    "message": "팝업 창 내 글씨와 아이콘의 크기를 조절합니다. 50-150% 사이 값을 입력하세요.",
+    "description": ""
+  },
+  "popup_element_scale_validation_error_value_format": {
+    "message": "정수 값을 입력하세요.",
+    "description": ""
+  },
+  "popup_element_scale_validation_error_value_range": {
+    "message": "50-150% 사이의 값을 입력하세요.",
+    "description": ""
+  },
+  "scale": {
+    "message": "비율",
+    "description": ""
+  },
+  "height": {
+    "message": "높이",
+    "description": ""
+  },
+  "width": {
+    "message": "너비",
+    "description": ""
+  },
+  "save": {
+    "message": "저장",
+    "description": ""
+  },
+  "saving": {
+    "message": "저장 중",
+    "description": ""
+  },
+  "saved_successfully": {
+    "message": "성공적으로 저장했습니다.",
+    "description": ""
+  },
+  "saved_error": {
+    "message": "설정 값을 저장하지 못했습니다. 다시 시도해주세요.",
+    "description": ""
+  },
+  "popup_size_validation_error_value_format": {
+    "message": "정수 값을 입력하세요.",
+    "description": ""
+  },
+  "popup_size_validation_error_value_range": {
+    "message": "높이는 200px에서 600px 사이, 너비는 400px에서 800px 사이 값을 입력해야 합니다.",
+    "description": ""
+  },
+  "tab_cleaner_validation_error_value_format": {
+    "message": "Input a positive integer.",
+    "modified_message": "Input a natural number.",
+    "message": "자연수를 입력하세요.",
+    "description": ""
+  },
+  "tab_cleaner_error_on_save": {
+    "message": "Please fix the error.",
+    "description": ""
+  },
```

Comment:
"tab_cleaner_error_on_save" is an error message displayed when saving the settings for the automatic tab clean feature. It prompts users to make corrections.

Reply:
Hmm, it's hard to test the error message... I translate this message based on the as-is version. But I think it would be better if the message gives more detailed information about why this error has been occurred.

Reply:
Thanks! I think it's fine as is because detailed errors are displayed for each form when this error is shown.